src/swarmClient/main.js

```diff
@@ -39,16 +36,6 @@ module.exports = {
         onclose = (onclose && once(onclose)) || (() => {});
 
 
-
-        if(public_pem) {
-            // throws an error if key is malformed
-            import_public_key_from_base64(public_pem);
-        }
-
-        public_pem = public_pem || pub_from_priv(private_pem);
```

Comment:
If I'm reading correctly, we used to generate a public key if only the private key was given. Now, we do not. Is this the case? This may also be a regression in startup behaviour.